### ISSUE TITLE
Powering off

### ISSUE THREAD

**Issue by oculos** [Xinyuan-LilyGO/TTGO-T-Display#62]
Hi,

I wonder if there's a combination (like connecting EN to GND) to disable the 3.3V regulator and turn the TTGO off. I know I could use deep sleep, but I'd rather cut all power from the device. Is that possible?

Does the TTGO have a EN pin?

**Comment by tadam777** [comment]
If you power from USB, there will always be power on the 3.3V
But I assume you're on battery, did you try the PWR_EN pin ? Its role isn't completely clear to me, but it looks like it commands a connection between the battery and the 3.3V regulator.

![image](https://user-images.githubusercontent.com/4885885/119467741-16094a00-bd46-11eb-9a5f-5d95ee31e43a.png)

**Comment by JFPayeur** [comment]
Set PWR_EN to 1 enable the battery voltage reading , but is consume a little tiny bit of power. It does nothing to power off the device

**Comment by tadam777** [comment]
Yes, it's at least 50µA through R27 (and maybe as much through R28), which can seem very low, but it's relatively high in comparison to the ESP32 sleep modes (deep sleep 10µA, hibernation 2.5µA).

**Comment by JFPayeur** [comment]
And It does power the "BAT" pin , 
And BAT power R6/R7 
![image](https://user-images.githubusercontent.com/11382177/191150886-3a9877e7-a72c-4dbd-8140-a01496d0ae76.png)
In the same order off magnitude of power loss
